### PR TITLE
Added a pull request template

### PR DESCRIPTION
**Pull request by yogeshj83**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## ISSUE(S):
+<!--- Links to JIRA tickets -->
+<!--- [FT-XXXX](link.to/FT-XXXX): Subject of FT-XXXX -->
+
+## SUMMARY
+<!--- Describe the change below -->
+
+<!--- include "Fixes #FT-XXX" if you have a relevant Jira ticket -->
```
Comment (nikhilbandi) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: We should make having a Jira issue compulsory.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<!--- delete section if not relevant -->
+
+## TESTING DONE
+#### Automated
+<!--- Please give link to teamcity build if there are any automated tests>
+<!--- that gets executed as a part of build.>
+#### Manual
+<!--- Please list down various use case which were part of manual testing.>
+
+#### Reviewers
```
Comment (nikhilbandi) [inline .github/PULL_REQUEST_TEMPLATE.md:37]: Will doing so automatically add these names are reviewers in the PR? If yes, great. If not, we can skip this.

**Review by nikhilbandi — CHANGES_REQUESTED** [review]
A couple minor comments / questions.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (yogeshj83) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Enforcement via PR templates is not possible. Please take a look at this discussion:
https://github.community/t/make-pr-template-field-mandatory/203194.
The main motive behind PR templates is to have uniformity across the PRs. Maybe while reviewing if the Jira field is empty, the reviewer could reject the PR.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (yogeshj83) [reply .github/PULL_REQUEST_TEMPLATE.md:37]: I don't think that PR templates could automatically populate the reviewer name but it does give the suggestion as you start typing the reviewer name starting with '@'.

**Comment by yogeshj83** [comment]
@nikhilbandi @anupbarve Can you review the changes?

**Review by anupbarve — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (nikhilbandi) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Got it.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (nikhilbandi) [reply .github/PULL_REQUEST_TEMPLATE.md:37]: OK.

**Review by nikhilbandi — APPROVED** [review]
Approving.

**Pull request merged**